ecs-init/logger/log.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The year seems to vary with files, and many files do not have it. Can the year be removed?

Reply:
sure! will remove it